assets/src/settings-page/index.js

```diff
@@ -120,6 +124,25 @@ function Root() {
 		return () => undefined;
 	}, [ didSaveOptions, downloadingTheme ] );
 
+	/**
+	 * Scrolls to the initially focused section after loading.
+	 */
+	useEffect( () => {
+		if ( fetchingOptions ) {
+			return;
+		}
+
+		if ( ! section ) {
+			return;
+		}
+
+		const focusedSection = document.getElementById( section );
+
+		if ( focusedSection ) {
+			focusedSection.scrollIntoView();
```

Comment:
Would smooth scrolling be desirable? 

```suggestion
			focusedSection.scrollIntoView( { behavior: 'smooth' } );
```

Reply:
Alternatively, for accessibility, I think what is needed is to actually `focus()` on the first focusable element. 

So this should be looking for the first `button` or `input` that is inside the `focusedSection` and then `focus()` on it. This will automatically scroll _and_ add the necessary keyboard focus.

Reply:
Done in `e41c971d9` with `select` elements also included